src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -56,8 +56,8 @@ public class ExtraNodeViewInfo
         public double Y;
         public bool ShowGeometry;
         public bool Excluded;
-        public bool IsInput;
-        public bool IsOutput;
+        public bool IsSetAsInput;
+        public bool IsSetAsOutput;
```

Comment:
API change?

Reply:
@ramramps This chass is introduced as part of Dynamo 2.0

Reply:
so, if we change this, we should let Alias team know this?

Reply:
Sure, although they most likely will not be using this public class

Reply:
I think so.. but we said if we changed the API, we will let them know of the new binaries..